### PR TITLE
#5029 fix: failed parsing of signed message

### LINKED ISSUE #5029: Failed parsing of signed message
Browser extension fails to parse signed message with attached signature

Screenshot 2023-03-20 at 14 38 15

@ioanmo226 please work on this issue after finishing your current one, I forwarded you email with attached `.eml`

### PR DESCRIPTION
**Pull request by ioanmo226**
This PR fixed failing parsing of signed message

close #5029 // if this PR closes an issue

----------------------------------

**Tests** _(delete all except exactly one)_:
- Tests added or updated

--------------------------------

### To be filled by reviewers

I have reviewed that this PR... _(tick whichever items you personally focused on during this review)_:
- [ ] addresses the issue it closes (if any)
- [ ] code is readable and understandable
- [ ] is accompanied with tests, or tests are not needed
- [ ] is free of vulnerabilities
- [ ] is documented clearly and usefully, or doesn't need documentation

### CODE REVIEW

**Comment by sosnovsky** [comment]
@ioanmo226 I checked email from original issue and we have this text `One attachment4` below message

Screenshot 2023-03-23 at 10 05 16

Can you please remove it? As we don't render any attachments there

**Comment by ioanmo226** [comment]
Which email do you mean? Customer email?
> I checked email from original issue

**Comment by sosnovsky** [comment]
Yes, the one from `.eml` file

File: extension/js/common/core/attachment.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@
 import { Buf } from './buf.js';
 import { Str } from './common.js';
 
-type Attachment$treatAs = 'publicKey' | 'privateKey' | 'encryptedMsg' | 'hidden' | 'signature' | 'encryptedFile' | 'plainFile';
+type Attachment$treatAs = 'publicKey' | 'privateKey' | 'encryptedMsg' | 'hidden' | 'signature' | 'encryptedFile' | 'plainFile' | 'inline_image';
```
[the remark this answers is not in the document]
Comment (ioanmo226) [reply extension/js/common/core/attachment.ts:8]: Yep!
Missed that one.

**Review by sosnovsky — APPROVED** [review]

**Pull request merged**